README.md

```diff
@@ -62,11 +62,8 @@
 
 ### Key Features <a name="key-features"></a>
```

Comment:
- [OPTIONAL] It would be better to remove the Client React.js and Express.js Server from the Tech Stack section since we're not using them. 🚮 

![image](https://user-images.githubusercontent.com/79058364/236119401-f4e1f0d9-4ae7-48c2-92ae-8d835f1230ef.png)